examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -218,6 +213,20 @@ export class TaskList extends DataObject implements ITaskList {
      * DataObject, by registering an event listener for changes to the task list.
      */
     protected async hasInitialized(): Promise<void> {
+        const saved = this.root.get<IFluidHandle<SharedMap>>("savedData");
+        if (!saved) {
```

Comment:
Always use strict null checks, here and also below for draftData (actually -- why didn't this fail the PR build...?  @Josmithr do you know?)
```suggestion
        if (saved === undefined) {
```